CONTRIBUTING.md

```diff
@@ -59,14 +62,29 @@ but there is a human involved there, so please forgive any delays.
 ## Contributing a feature
 
 In order to contribute a feature to Istio you'll need to go through the following steps:
-- Discuss your idea with the appropriate [working groups](WORKING-GROUPS.md).
+
+- Discuss your idea with the appropriate [working groups](WORKING-GROUPS.md) on the working 
+group's mailing list.
+
 - Once there is general agreement that the feature is useful, create a GitHub issue to track the discussion. The issue should include information
-about the requirements and use cases that it is trying to address.
-- Include a discussion of the proposed design and technical details of the implementation in the issue.
-- If the feature is substantial enough, create a design document as outlined in the previous section. Link this design document within
-the GitHub issue.
-- Once there is general agreement on the technical direction, submit a PR.
-- Contribute documentation of the feature to the [istio.io](https://istio.io) repo (https://github.com/istio/istio.github.io).
+about the requirements and use cases that it is trying to address. Include a discussion of the proposed design and technical details of the
+implementation in the issue.
+
+- If the feature is substantial enough:
+
+  - Working group leads will ask for a design document as outlined in the previous section.
+  Create this document and add a link to it in the GitHub issue. Don't forget to send a note to the 
+  working group to let everyone know your document is ready for review.
+
+  - Depending of the breath of the design and how contentious it is, the working group leads may decide
+  the feature needs to be discussed in one or more working group meetings before being approved.
+
+  - Once the major technical issues are resolved and agreed upon, post a note to the working group's mailing
+  list with the design decision and the general execution plan.
+
+- Submit a PR to istio/istio with your code changes.
```

Comment:
one or more PRs?  Something about breaking into manageable chunks? Or too much detail here?

Could just say:

Submit PRs to istio/istio with your code changes.

Maybe same with documentation (s/a PR/PRs/)